src/components/Pagination.tsx

```diff
@@ -20,6 +21,10 @@ type Props = {
   totalCount: number;
   totalPages: number;
   keyword: string;
+  pageData: MainTableRow[];
+  itemsPerPage: number;
+  totalItems: number;
+  updateSort: (sort?: string | null) => void;
```

Comment:
I have a feeling these should be set as optional. Or at least, my IDE complains if they're not.
```suggestion
  pageData?: MainTableRow[];
  itemsPerPage?: number;
  totalItems?: number;
  updateSort?: (sort?: string | null) => void;
```